ALiAS_FAQs.md

```diff
@@ -0,0 +1,46 @@
+                                                FAQs
+Q1. What is ASET AliAS?
+Ans. ALiAS is not your regular Amity Club. ALiAS is a community, founded in 2010, which was developed with the aim to promote the use of Linux in the campus and get students familiar with the world of Open Source. The community members help students with their doubts pertaining to tech, conduct workshops, webinars,organize meet-ups (online and in-person) and most important get you connected.
+
+Q2. Why should I join ALiAS?
+Ans. ALiAS is by far, the largest open-source community in Amity University. We are
+open to developers not only inside the campus, but also from various places. We have got assistance
+from alumni and people from tech, near and far. We have got a presence in communities like PyDelhi, ILUG-D, LinuxChix India, Mozilla Delhi, etc. and can help you get connected.
+
+Q3. How is the community of AliAS?
```

Comment:
No question is stupid. 
@sachanakshat I am not sure how this would be appropriate as a FAQ. Have you ever seen an insurance company or conference writing about themselves as a personal opinion in FAQ?  
This question is loaded. Please remove.

Reply:
+1